components/SearchResults/index.js

```diff
@@ -104,43 +137,132 @@ export default function SearchResults() {
   return (
     <>
       <div>
-        for input: <b>{searchQuery}</b>
+        searchresults for input: <StyledBoldText>{searchQuery}</StyledBoldText>
+        <StyledSearchStats>
+          <i>
+            Name: <StyledBoldText>{cardByName.length}</StyledBoldText>
+          </i>
+          <i>
+            Description: <StyledBoldText>{cardByDesc.length}</StyledBoldText>
+          </i>
+          <i>
+            Meaning reversed:
+            <StyledBoldText>{cardByMeaningRev.length}</StyledBoldText>
+          </i>
+          <i>
+            Meaning upside:
+            <StyledBoldText>{cardByMeaningUp.length}</StyledBoldText>
+          </i>
+        </StyledSearchStats>
+        <select
+          onChange={(e) => renderResults(e)}
+          style={{
```

Comment:
Maybe you could avoid using inline styling here?

Reply:
tried it, but it won't apply : 
![image](https://user-images.githubusercontent.com/115098194/234222499-d5a0288a-aaeb-42d7-bfaf-5d859c18c724.png)

### i will implement this idea at a later point : [LINK](https://plainenglish.io/blog/custom-select-component-in-react-and-styled-components-e1f3b0a3a287)